### DISCUSSION TITLE
matplotlib 3.6.1

### DISCUSSION THREAD

**Discussion by petsuter** [spectrochempy/spectrochempy#483] [Q&A] [answered]
I get this error with matplotlib 3.6.1 for `from spectrochempy.core.plotters.multiplot import multiplot`:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "Python\lib\site-packages\spectrochempy\core\processors\baseline.py", line 21, in <module>
    from spectrochempy.core.plotters.multiplot import multiplot
  File "Python\lib\site-packages\spectrochempy\core\plotters\multiplot.py", line 26, in <module>
    from matplotlib.tight_layout import (
ImportError: cannot import name 'get_renderer' from 'matplotlib.tight_layout' (Python\lib\site-packages\matplotlib\tight_layout.py)
```
It works with matplotlib 3.5.1

**Comment by atravert** [comment, marked as answer]
Thank you. Yes we have noticed this problem with matplotlib 3.6.0 (https://github.com/spectrochempy/spectrochempy/pull/481#issuecomment-1263680110). As a quick fix we have changed the requirements form `matplotlib>=3.5` to `matplotlib=3.5`.
We will look at how to makes scpy compatible with matplotlib 3.6.0 i nthe near future.

Best,
Arnaud
